### PR TITLE
Add nonstd-span-lite 0.5.0 release

### PR DESCRIPTION
**Pull request by mattyclarkson**
How do I test this before we merge?

When this merges, I'll add all the other `nonstd-*` in a similar fashion.

### CODE REVIEW

**Comment by mattyclarkson** [comment]
@sarum9in, RFC

**Comment by mattyclarkson** [comment]
@sarum9in what do you think? Could we get this in as a first step?

File: upstream.wrap [hunk 1]
```diff
@@ -0,0 +1,6 @@
+[wrap-file]
+directory = nonstd-span-lite-0.5.0
```
Comment (sarum9in) [inline upstream.wrap]: It seems like the directory name is `span-lite-0.5.0` according to
```
$ tar tf v0.5.0 
span-lite-0.5.0/
...
```

**Review by sarum9in — CHANGES_REQUESTED** [review]

File: upstream.wrap (hunk 1 above)
Comment (mattyclarkson) [reply upstream.wrap]: Thanks! I actually thought that determined what folder the wrap unpacked into. Thanks for the clarification :smile:

**Pull request merged**